### ISSUE TITLE
Create a reference implementation of Automated Threat Actor Classification (for demonstration purposes)

### ISSUE THREAD

**Issue by rhohimer** [oasis-open/tac-ontology#53]
To help others see the business value of using graph technology to automate SME logic.

**Comment by rhohimer** [comment]
I needed to work on the STIX 2.1 JSON to STIX 2.1 Ontology conversion in order to proceed with this issue.

**Comment by rhohimer** [comment]
I have been working this issue. I now have 99 percent of the apt1.json example converting to apt1.ttl

Key requirements:

- Conversion
    - STIX 2.1 JSON to STIX.OWL to enable STIX JSON Reports to be converted to STIX Turtle Reports
- Characteristic Recognition
    - Illustrate **object recognition** by seeing *_defining characteristics_*
    - Illustrate Description Logics reasoning about defining characteristics

**Comment by rhohimer** [comment]
I've pushed a new branch to the repository: issue-53-reference-implementation.

There is a new utilities directory that contains the JSON-LD context conversion method and the SPARQL-Anything conversion method.  I have elected to pursue the SPARQL-Anything approach but left the context conversion method in case others wish to continue developing it.

There is a new third-party extension. The Healthcare Agent Library contains an ontology (hal-library.owl) that extends the Threat Agent Library. The hal-example.owl file illustrates subclassing of a tal:RecklessEmployee with a hal:CuriousHeathcareWorker

We'll need to make progress on other issues to be ready for the Borderless Cyber conference.
